container/shim/src/index.js

```diff
@@ -120,6 +125,8 @@ if (cluster.isPrimary) {
       controller.abort()
     }, GATEWAY_TIMEOUT)
 
+    // Use node-fetch or undici instead, the `"error"`
+    // event makes this difficult to consume
```

Comment:
I didn't know which way you wanted to go, but I think it is important to use a promise native request library.